### PR TITLE
[FIXED JENKINS-51332] Whitelist Calendar.MONDAY/JANUARY et al

### PR DESCRIPTION
**Pull request by abayer**
[JENKINS-51332](https://issues.jenkins-ci.org/browse/JENKINS-51332)

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist [hunk 1]
```diff
@@ -114,20 +114,39 @@ new java.text.SimpleDateFormat java.lang.String
...
 staticField java.util.Calendar MONTH
+staticField java.util.Calendar NOVEMBER
+staticField java.util.Calendar OCTOBER
+staticField java.util.Calendar SATURDAY
 staticField java.util.Calendar SECOND
+staticField java.util.Calendar SEPTEMBER
+staticField java.util.Calendar SUNDAY
+staticField java.util.Calendar THURSDAY
+staticField java.util.Calendar TUESDAY
+staticField java.util.Calendar WEDNESDAY
```
Comment (dwnusbaum) [inline src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: No `UNDECIMBER`? Half-joking, but maybe it makes sense to add it for consistency?

**Comment by dwnusbaum** [comment]
Two other fields that seem like they might be used often enough to add are `Calendar.AM` and `Calendar.PM`, especially since we already whitelisted `Calendar.AM_PM`.

File: src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist (hunk 1 above)
Comment (abayer) [reply src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: wat. =)

**Comment by abayer** [comment]
Good call on `AM` and `PM` - added.

File: src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist (hunk 1 above)
Comment (dwnusbaum) [reply src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: https://stackoverflow.com/questions/44716517/what-does-calendar-undecimber-do 😄

Comment (abayer) [reply src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: I'm still saying wat and running away from that. =)

**Pull request merged**